src/runtime/windows_d3d12compute_x86.cpp

```diff
@@ -2,4 +2,12 @@
 // located in "d3d12compute.cpp"; this wrapper is used as a mean to "decorate" the
 // module such that the proper build flags and target triple can be determined based
 // on the prefix (windows_) and suffix (_x86) when building the runtime module.
+
```

Comment:
is there any reason we shouldn't just rename `d3d12compute.cpp` instead of including it? (if so, let's do it as a followup PR)

Reply:
I think the intent is to have `windows_d3d12compute_arm64.cpp` shortly.

Reply:
Yeap, it was briefly discussed here:
https://github.com/halide/Halide/issues/5156#issuecomment-674248591